mod/quiz/accessrule/seb/amd/src/validate_keys.js

```diff
@@ -0,0 +1,99 @@
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Render mustache template examples within the component library.
+ *
+ * @module     quizaccess_seb/validate_keys
+ * @author     Andrew Madden <andrewmadden@catalyst-au.net>
+ * @copyright  2021 Catalyst IT
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+import Ajax from 'core/ajax';
+import Notification from "core/notification";
+import * as View from 'quizaccess_seb/view';
+
+// SafeExamBrowser object will be automatically initialized if using the SafeExamBrowser application.
+window.SafeExamBrowser = window.SafeExamBrowser || null;
+
+/** @var loadingAlert Reference to modal pop-up. Defined at module-level scope. */
+let loadingAlert;
+
+/**
+ * Once the keys are fetched, action checking access.
+ */
+const safeExamBrowserKeysUpdated = async () => {
+
+    // Display a message to user that a background check for SEB access is running.
+    loadingAlert = await Notification.alert(
+        'Safe Exam Browser requirement',
```

Comment:
Would be good to replace those hard-coded strings

Reply:
No longer relevant.